docs/tutorials/infrastructure/binary/index.md

```diff
@@ -0,0 +1,431 @@
+- To enable HTTPS in the browser, you will need SSL certificates and your own domain name pointing to the server.
+
+## Obtaining the Temporal Binaries
+
+You'll begin by downloading and configuring the Temporal Server binaries.
+
+The Temporal Core Server can be obtained from its [Github Releases Page](https://github.com/temporalio/temporal/releases/). Download the newest binary for your operating system (probably `linux_amd64`) and extract it on the command line using `curl` and `tar`:
+
+```bash
+curl -OL https://github.com/temporalio/temporal/releases/download/v1.24.2/temporal_1.24.2_linux_amd64.tar.gz
+tar -xzf temporal_1.24.2_linux_amd64.tar.gz
+```
+
+The extracted binary will be called `temporal-server`. Move it to a directory on your path like `/usr/bin` and make it executable:
+
+```bash
+sudo mv temporal-server /usr/bin/temporal-server
+sudo chmod +x /usr/bin/temporal-server
+```
+
+You'll also need the Temporal UI server. The Temporal UI Server is a standalone binary that can also be obtained from its [Github Releases page](https://github.com/temporalio/ui-server/releases). Download the newest binary for your operating system and extract it on the command line using `curl` and `tar`:
+
+```bash
+curl -OL https://github.com/temporalio/ui-server/releases/download/v2.28.0/ui-server_2.28.0_linux_amd64.tar.gz
+tar -xzf ui-server_2.28.0_linux_amd64.tar.gz
+```
+
+The extracted binary will be called `ui-server`. Move it to a directory on your path like `/usr/bin`:
+
+```bash
+sudo mv ui-server /usr/bin/temporal-ui-server
+sudo chmod +x /usr/bin/temporal-ui-server
+```
+
+At this point, you've downloaded everything you need. The last thing to do is create a `temporal` user on your server that has the appropriate permissions to run the Temporal Service, and directory accessible to this user to store your data in. Run the following commands:
+
+```bash
+sudo useradd temporal
+sudo mkdir /etc/temporal
+sudo chown temporal /etc/temporal
+```
+
+Next, you'll create configuration files for both the Temporal Server and the UI Server in the `/etc` directory.
+
+## Configuring the Temporal Binaries
+
+In this tutorial, you'll configure a connection to a SQLite database, since it doesn't require any additional dependencies. Using your favorite text editor, open a new file called `/etc/temporal-server.yaml`:
+
+```bash
+sudo vim /etc/temporal-server.yaml
+```
+
+Paste the following contents into the file for a starting configuration. You can update any of these values later.
+
+```
+log:
+  stdout: true
+  level: info
+
+persistence:
+  defaultStore: sqlite-default
+  visibilityStore: sqlite-visibility
+  numHistoryShards: 1
```

Comment:
dev-server uses 4 hard-coded here. I think it may be a bit academic because SQLite doesn't allow concurrent writes, but I wouldn't take it lower than 4 anyway. A real production system would generally not have this lower than 256, probably 512.